collections/hpc/roles/slurm/README.md

```diff
@@ -24,15 +24,49 @@ Note: this role requires *bluebanquise_filters* package to be installed.
 ### Munge key
 
 **IMPORTANT**: before using the role, first thing to do is to generate a
-new munge key file. To do so, generate a new munge.key file using:
+new munge key file. This file need to be the same on all nodes of the Slurm cluster, and so need to be spread by the role.
 
-```bash
-dd if=/dev/urandom bs=1 count=1024 > munge.key
-mungekey -c -k /etc/bluebanquise/roles/community/slurm/files/munge.key
+The role do not provide default munge key file, as it is considered a security risk.
+
+The role supports currently 2 ways to provide key file:
+
+* Provide plain file into `files` subfolder of executed playbook folder
+* Provide base64 encoded string
+
+First step for both methods is to generate a new munge.key file locally using (note that you need munge package to be installed on your system):
+
+```
+mungekey -c -k munge.key
+```
+
+Then, next step depends of method choose.
+
+#### Provide plain file
+
+Assuming your playbooks folder is `$HOME/playbooks`, create a `files` subfolder, and place your key inside:
+
+```
+mkdir -p $HOME/playbooks/files
+mv munge.key $HOME/playbooks/files/munge.key
+```
+
+Role will spread this file across all nodes of the Slurm cluster.
+
```

Comment:
You may add a note about encrypting the file with [ansible-vault](https://docs.ansible.com/ansible/latest/vault_guide/vault_encrypting_content.html#encrypting-files-with-ansible-vault) if the file has to be included in a version control repository

Reply:
Good point !
I will add that this evening 😊